package.json

```diff
@@ -32,14 +32,18 @@
   "author": "Josh Perez <josh.perez@airbnb.com>",
   "license": "MIT",
   "devDependencies": {
-    "@kadira/storybook": "^2.35.3",
+    "@storybook/addon-actions": "^5.1.11",
+    "@storybook/addon-links": "^5.1.11",
+    "@storybook/addons": "^5.1.11",
+    "@storybook/react": "^5.1.11",
     "acorn": "^6.3.0",
     "acorn-jsx": "^5.0.1",
     "airbnb-browser-shims": "^3.2.1",
     "aphrodite": "^2.3.1",
     "babel-cli": "^6.26.0",
     "babel-core": "^6.26.3",
     "babel-jest": "^21.2.0",
+    "babel-loader": "^7.1.5",
```

Comment:
Was this installed by the storybook script or by me? 🤔

Reply:
Storybook doesn't work without it.

Reply:
yeah, it's a storybook dependency https://github.com/storybookjs/storybook/blob/next/package.json#L150